sonar-python-plugin/src/main/resources/org/sonar/plugins/python/bandit/rules.json

```diff
@@ -0,0 +1,367 @@
+  },
+  {
+    "key": "B309",
+    "name": "B309 blacklist httpsconnection",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b309-httpsconnection"
+  },
+  {
+    "key": "B310",
+    "name": "B310 blacklist urllib_urlopen",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b310-urllib-urlopen"
+  },
+  {
+    "key": "B311",
+    "name": "B311 blacklist random",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b311-random"
+  },
+  {
+    "key": "B312",
+    "name": "B312 blacklist telnetlib",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b312-telnetlib"
+  },
+  {
+    "key": "B313",
+    "name": "B313 blacklist xml_bad_celementtree",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b313-b320-xml"
+  },
+  {
+    "key": "B314",
+    "name": "B314 blacklist xml_bad_elementtree",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b313-b320-xml"
+  },
+  {
+    "key": "B315",
+    "name": "B315 blacklist xml_bad_expatreader",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b313-b320-xml"
+  },
+  {
+    "key": "B316",
+    "name": "B316 blacklist xml_bad_expatbuilder",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b313-b320-xml"
+  },
+  {
+    "key": "B317",
+    "name": "B317 blacklist xml_bad_sax",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b313-b320-xml"
+  },
+  {
+    "key": "B318",
+    "name": "B318 blacklist xml_bad_minidom",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b313-b320-xml"
+  },
+  {
+    "key": "B319",
+    "name": "B319 blacklist xml_bad_pulldom",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b313-b320-xml"
+  },
+  {
+    "key": "B320",
+    "name": "B320 blacklist xml_bad_etree",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b313-b320-xml"
+  },
+  {
+    "key": "B321",
+    "name": "B321 blacklist ftplib",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b321-ftplib"
+  },
+  {
+    "key": "B322",
+    "name": "B322 blacklist input",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b322-input"
+  },
+  {
+    "key": "B323",
+    "name": "B323 blacklist unverified_context",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html#b323-unverified-context"
+  },
+  {
+    "key": "B324",
+    "name": "B324 blacklist hashlib_new_insecure_functions",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html"
+  },
+  {
+    "key": "B325",
+    "name": "B325 blacklist tempnam",
+    "url": "https://docs.openstack.org/bandit/latest/api/bandit.blacklists.html"
```

Comment:
https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html#b325-tempnam ?

Reply:
actually, I think it's better to have all these B3XX issues link to their corresponding issue on https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html and https://bandit.readthedocs.io/en/latest/blacklists/blacklist_imports.html since it's the official doc, not an external website...